Vizzuality/forest-atlas-landscape-cms
# Add/delete fields from MapBuilder form

This PR removes INCLUDEMEASUREMENTTAB and FORMA and adds customColorTheme and useWebmapBasemap to map settings page.

## Testing instructions

1. Open a site with pages of type map or create a new map page.
2. Go to the map tab/step
3. Removed fields should be gone and added fields should appear.

## Screenshot
![image](https://user-images.githubusercontent.com/268405/67965308-bc887a00-fbf9-11e9-9370-cba1a89c2254.png)
## Pivotal Tracker

[Task](https://www.pivotaltracker.com/story/show/169190629)
